store/postgres/src/fork.rs

```diff
@@ -220,7 +221,11 @@ query Query ($id: String) {{
             map
         };
 
-        Ok(Some(schema.make_entity(map)?))
+        Ok(Some(
+            schema
+                .make_entity(map)
+                .map_err(|e| StoreError::Unknown(anyhow!(e)))?,
```

Comment:
I think it's probably worth adding something like an EntityValidationError value 

something along the lines of 

#[error("Entity validation failed: {0}")}
EntityValidationError(#[from] graph::whatever::EntityValidationError)

Reply:
since we do know what kind of error it is in this case

Reply:
changed